packages/hash/api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -183,6 +189,198 @@ export const getAllLatestEntitiesResolver: ResolverFn<
   return entitySubgraph as Subgraph;
 };
 
+export const aggregateEntitiesResolver: Extract<
+  QueryResolvers<LoggedInGraphQLContext>["aggregateEntities"],
+  Function
+> = async (
+  _,
+  {
+    operation,
+    constrainsValuesOn,
+    constrainsPropertiesOn,
+    constrainsLinksOn,
+    constrainsLinkDestinationsOn,
+    isOfType,
+    hasLeftEntity,
+    hasRightEntity,
+  },
+  { dataSources },
+  __,
+) => {
+  const { graphApi } = dataSources;
+
+  /** @todo - should we throw an error, or should we just ignore the constraint and hope for the best */
+  if (
+    (operation.pageNumber !== undefined && operation.pageNumber !== null) ||
+    (operation.itemsPerPage !== undefined && operation.itemsPerPage !== null)
+  ) {
+    throw new ApolloError(
+      "Pagination on aggregateEntities results is not currently supported",
+    );
+  }
+
+  if (operation.multiSort !== undefined && operation.multiSort !== null) {
```

Comment:
doesn't `!= null` cover both `null` and `undefined` or does our linting not allow this? (applies for the other similar checks in this file)

Reply:
It does cover that, but I know other devs on the team don't like that check, and I think they would have recommended I do `!operation.multiSort`, however I think it's better here to be explicit. We're not doing a whole falsey check, we're just checking it's not null or undefined.

Reply:
It does, and our linting allows it. I would tend to use `x != null` if I was just writing code for myself, but it relies on people knowing this not exactly obvious behavior. 

We currently leave it up to personal preference – if I had to choose one to enforce I'd probably go for the  longer formulation shown in this diff, as it is clearer and less distracting (I would prioritise clarity over verbosity).